RoddyCore/src/de/dkfz/roddy/core/Analysis.groovy

```diff
@@ -243,7 +235,7 @@ class Analysis {
             }
 
             if (!test && !canStartJobs(ds)) {
-                logger.postAlwaysInfo("The " + Constants.PID + " " + ds.getId() + " is still running and will be skipped for the process.")
+                logger.postAlwaysInfo("The ${Constants.DATASET_HR} ${ds.id} is still running and will be skipped for the process.")
```

Comment:
As I said, "dataset" in the conventional way is fine. "dataset" as identifier should be modelled as constant.